src/shared/components/copy/Copy.module.css

```diff
@@ -4,6 +4,7 @@
   padding: 0 14px;
   align-items: center;
   justify-content: center;
+  font-size: var(--var-copy-font-size);
```

Comment:
1) Why use `var` as part of the variable name?
2) If you want to make font size of this component customisable, provide the default value (`font-size: var(--copy-button-font-size, 12px);`. Confirm with Andrea whether she plans to always use 12px for this element.
3) Unless parent component needs to override this style rule, there's no need to define it in the parent component (see your changes in VariantVCF.module.css)